openshift/templates/cakephp-mysql.json

```diff
@@ -354,6 +354,12 @@
   ],
   "parameters": [
     {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the objects defined in this template.",
```

Comment:
it's not all of the objects, it's all the frontend objects.

the db-related objects need their names parameterized as well, so you can actually instantiate this template multiple times in the same namespace w/ no collisions (assuming you pick unique parameter values)